pom.xml

```diff
@@ -1901,20 +1900,6 @@
             </properties>
         </profile>
 
-        <profile>
```

Comment:
I'm not sure if it's good to remove the Spark Hadoop version profile. The Spark master branch has already moved to Hadoop 3.3.1. So if we upgrate Spark to 3.3 we still need the profile.

Reply:
when we update spark

before: we update 2 parameters `spark.version` and `hadoop.binary.version`, and we need to ensure in our mind that the `spark.archive.name` is also need to update.

now: we update 1 parameter `spark.version` only or sometimes 2 if `spark.archive.name` changes its static pattern

Reply:
I see, then I'm fine with this change. We have already use `spark.archive.name` in ga workflow, so the Spark Hadoop version can be removed.